petab/core.py

```diff
@@ -159,16 +159,15 @@ def get_optimization_parameters(self):
         return get_optimization_parameters(self.parameter_df)
 
     def get_dynamic_simulation_parameters(self):
-        return get_dynamic_simulation_parameters(
-            self.sbml_model, self.parameter_df)
+        return get_model_parameters(self.sbml_model)
 
     def get_dynamic_parameters_from_sbml(self):
         """
         Provide list of IDS of parameters which are dynamic, i.e. not
         fixed.
         See get_dynamic_parameters_from_sbml.
         """
-        return get_dynamic_parameters_from_sbml(self.sbml_model)
+        return get_model_parameters(self.sbml_model)
```

Comment:
delete whole method since doing the same as the one above?

Reply:
Wasn't sure which one to remove. This is one it is.